build.gradle

```diff
@@ -162,7 +162,7 @@ dockerCompose {
     ]
     useComposeFiles = ["${project.rootDir.path}/docker/compose.yml"]
     waitForHealthyStateTimeout = Duration.ofMinutes(3)
-    waitForTcpPorts = false
+    waitForTcpPorts = true
```

Comment:
Setting this to true should avoid race condition issue when postgres container starts up and immediately followed by tx-exclusion-api integration test, given the healthy state check might not be good enough to ensure that the postgres container is ready to accept connection, especially when the tx-exclusion-api integration test only requires postgres container alone.